package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@wpsyntex/polylang",
```

Comment:
This lines should be in compliance with the current plugin.

Reply:
Sure! And also put its version to `1.0.0` instead

Done in 9991b959c14ff770602a359843938343a099867e